Sources/GraphQLTransportWS/Server.swift

```diff
@@ -18,6 +18,8 @@ public class Server<InitPayload: Equatable & Codable> {
     
     var auth: (InitPayload) throws -> Void = { _ in }
     var onExit: () -> Void = { }
+    var onOperationComplete: () -> Void = {}
+    var onOperationError: () -> Void = {}
```

Comment:
It would be convenient for both of these to provide the ID of the operation that has completed or errored.

Reply:
Good call, added.